docs/conf.py

```diff
@@ -34,32 +34,23 @@ def setup(app):
 
 conf = ConfigParser()
 
-sys.path.insert(0, os.path.abspath('../'))
 sys.path.insert(0, os.path.abspath('hstaxe/'))
 sys.path.insert(0, os.path.abspath('exts/'))
 
 # -- General configuration ---------------------------------------------------
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
 
 # General information about the project
-package = importlib.import_module(setup_cfg['name'])
-project = setup_cfg['name']
-author = setup_cfg['author']
+package = 'hstaxe'
+project = 'hstaxe'
+author = 'STScI'
 copyright = '{0}, {1}'.format(datetime.datetime.now().year, author)
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-try:
-    version = package.__version__.split('-', 1)[0]
-    # The full version, including alpha/beta/rc tags.
-    release = package.__version__
-except AttributeError:
-    version = 'dev'
-    release = 'dev'
+version = release = '1.0.1'
```

Comment:
@rosteen , since `docs/conf.py` cannot read generated version now, you will have to remember to bump this when you do a release. FYI.

Reply:
Yup, I noticed that. Hopefully will only happen rarely (or once, when we fix the bugs we have on our plate).